src/App.tsx

```diff
@@ -1,61 +1,134 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import 'bulma/css/bulma.css';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import { getUserById, findUserById } from './helpers/helpers';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [isErrorOnUserSelect, setErrorOnUserSelect] = useState(false);
+  const [isErrorOnTitleInput, setErrorOnTitleInput] = useState(false);
+  const [todos, setTodos] = useState(getUserById);
+
+  const removeSpacesFromTittle = title.trim();
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.currentTarget.value);
+    setErrorOnTitleInput(false);
```

Comment:
Do you need to trigger this function if `isErrorOnUserSelect` has already been `false` ?